runtimes/pythonrt/runner/tests/main_test.py

```diff
@@ -25,22 +26,28 @@
 import values
 
 
+def new_test_runner(code_dir, worker=None, server=None):
+    runner = main.Runner(
+        id="runner1",
+        worker=worker,
+        code_dir=code_dir,
+        server=server,
+    )
+    runner._inactivty_timer.cancel()
```

Comment:
is this the gist of this pr? not sure i understand - there is a lingering timer?

Reply:
Same. But also FYI there's a typo in the timer name.

Reply:
Fixed, thanks!